docs/section21/Rule21-4.md

```diff
@@ -0,0 +1,45 @@
+
+# Boiler - Rule 21-4  
+
+**Rule ID:** 21-4  
+**Rule Description:** When baseline building does not use purchased heat, baseline systems 1,5,7,11,12 shall be modeled with natural draft boilers.  
+**Rule Assertion:** B-RMR  
+**Appendix G Section:** Section 21 Boiler  
+**Appendix G Section Reference:** Section G3.1.3.2 Building System-Specific Modeling Requirements for the Baseline model  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
```

Comment:
do we also need the baseline system type = 1, 5, 7, 11 (except CZ 0-3a), 12, 1a, 7a, 11a (expect cz 0-3a) check over here?

Reply:
I agree - it should be system type 1,5,7,11,12, 1a, 7a, 11a and 12a. 

Also, I would consider assigning different letters to Systems 11 and 11a in CZ 0-3a vs all other CZ. For example, 11.1 
 would be for System 11 w/electric resistance (CZ 0-3a) and 11.2 would be System 11 w/boiler, for all other CZs.